transactional_licence_form/tests/test_tlf_utils.py

```diff
@@ -0,0 +1,146 @@
+import unittest
+from unittest import mock
+
+from transactional_licence_form import utils
+
+
+class TestTLFUtils(unittest.TestCase):
+    @mock.patch("transactional_licence_form.utils.settings")
+    @mock.patch("transactional_licence_form.utils.EmailSender.__new__")
+    @mock.patch("transactional_licence_form.utils.sanitize_and_format_response_as_xml")
+    def test_send_form_response(self, mock_sanitize, mock_email_sender, mock_settings):
+        from_email = "from@example.com"
```

Comment:
Probably worth sorting this into AAA? Could add comments to break up the arranges, because I understand there's a lot of them!